IoTCoreDefaultApp - Dutch language added (nl-NL) see #236

Vinz68 opened the pull request:
Contribute IoTCoreDefaultApp - Dutch language (nl-NL)

First time ever for me to use & contribute via Git / GitHub ; 
so positive feedback to do this contribution better in the future is very welcome... thanks.

Vinz68 committed:
Create Resources.resw

Add support for Dutch (nl-NL)

Vinz68 committed:
Update IoTCoreDefaultApp.csproj

Added Dutch (nl-NL) string resources (issue #236)

msftclas commented:
Hi **@Vinz68**, I'm your friendly neighborhood Microsoft Pull Request Bot (You can call me MSBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.microsoft.com.
    </span>

TTYL, MSBOT;

msftclas commented:
**@Vinz68**, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, MSBOT;

Vinz68 committed:
Review comments processed. 

Some textual changes in Dutch (nl-NL; spelling and an untranslated label and disclaimer)

bfjelds commented:
Our localization team ran the resw through our tools and it looks great.  Thanks!
